lemmatize_text_data_WIP.py

```diff
@@ -0,0 +1,43 @@
+from settings import DATA_DIR, MODEL_DIR
+import os
+
+import pandas as pd
+from morphodita import get_lemmas
+
+
+def get_lemmatized_df(dataframe: pd.DataFrame, tagger_file):
+    """Get lemmatized text data from the dataframe."""
+
+    name_lemmatized = get_lemmas(tagger_file, dataframe["project_name"].tolist())
+    description_lemmatized = get_lemmas(
+        tagger_file, dataframe["project_description"].tolist()
+    )
+    interest_lemmatized = get_lemmas(tagger_file, dataframe["public_interest"].tolist())
+    df = pd.DataFrame(
+        {
+            "project_name": name_lemmatized,
+            "project_description": description_lemmatized,
+            "public_interest": interest_lemmatized,
+            "status": dataframe["status"],
+        }
+    )
+
+    df["status"].replace(
```

Comment:
this mapping could be put to separate file to be easily founded -- and even README can link that file

Reply:
do you mean a different file than settings.py?